mk/dev.mk

```diff
@@ -1,7 +1,9 @@
 KUMA_DIR ?= .
 ENVOY_VERSION = $(word 5, $(shell ${KUMA_DIR}/tools/releases/version.sh))
+CI_TOOLS_VERSION = dev
```

Comment:
This will be annoying to maintain (needs to be updated when branching). Could we use something with `version.sh`?